src/System.IO.FileSystem.Watcher/src/Resources/SR.cs

```diff
@@ -1,8 +1,4 @@
-// This is auto generated file. Please don’t modify manually.
-// The file is generated as part of the build through the ResourceGenerator tool 
-// which takes the project resx resource file and generated this source code file.
-// By default the tool will use Resources\Strings.resx but projects can customize
-// that by overriding the StringResourcesPath property group.
+// Do not edit this file manually it is auto-generated during the build based on the .resx file for this project.
```

Comment:
FYI This broken the TFS build so I submitted this change on the TFS side and I expect it to be mirrored out.